Intro to Ethereum developer docs: explicitly state ether === ETH (#4513)
ethereum/ethereum-org-website: src/content/developers/docs/intro-to-ethereum/index.md

Cryptographic mechanisms ensure that once transactions are verified as valid and added to the blockchain, they can't be tampered with later. The same mechanisms also ensure that all transactions are signed and executed with appropriate "permissions" (no one should be able to send digital assets from Alice's account, except for Alice herself).

> ## What is ether(ETH)? {#what-is-ether}

Reviewer: No need to add here, we are explaining below :-)
```suggestion
## What is ether? {#what-is-ether}
```

> The purpose of ether which is denoted as ETH, the cryptocurrency, is to allow for a market for computation. Such a market provides an economic incentive for participants to verify and execute transaction requests and provide computational resources to the network.

Reviewer: ```suggestion
**Ether (ETH)** is the native cryptocurrency of Ethereum. The purpose of ether is to allow for a market for computation. Such a market provides an economic incentive for participants to verify and execute transaction requests and provide computational resources to the network.
```
